.ci/Jenkinsfile

```diff
@@ -197,19 +198,20 @@ def runBenchmark(version){
   }
 }
 
-def runJob(rubyVersion){
+def runJob(rubyVersion, boolean isCoverage=false){
   def branch = env.BRANCH_NAME
 
   if (env.CHANGE_ID) {
     branch = env.CHANGE_TARGET
   }
-  
-  build( job: "apm-agent-ruby/apm-agent-ruby-downstream/${branch}", 
+
+  build( job: "apm-agent-ruby/apm-agent-ruby-downstream/PR-485",
```

Comment:
This is only required for testing purposes as it does use the master branch as the target branch by default.

Reply:
```suggestion
  build( job: "apm-agent-ruby/apm-agent-ruby-downstream/${env.BRANCH_NAME}",
```